internal/mage/util/engine.go

```diff
@@ -202,10 +210,11 @@ func devEngineContainer(c *dagger.Client, arch string, version string, opts ...D
 	if err != nil {
 		panic(err)
 	}
-	return c.Container(dagger.ContainerOpts{Platform: dagger.Platform("linux/" + arch)}).
+
+	container := c.Container(dagger.ContainerOpts{Platform: dagger.Platform("linux/" + arch)}).
 		From("alpine:"+alpineVersion).
 		WithExec([]string{
-			"apk", "add", "--no-cache",
+			"apk", "add",
```

Comment:
Why are we removing the `--no-cache` option?

Reply:
Good spot, might have missed this one during the rebase, will add a commit to keep that flag.

Reply:
Fixed this